app/src/main/java/com/cpr3663/cpr_scouting_app/Logger.java

```diff
@@ -202,22 +202,26 @@ public void LogEvent(int in_EventId, float in_X, float in_Y, boolean in_NewSeque
                 seq_number_prev_common = ++seq_number;
         }
 
-        String prev="";
+        String prev = "";
         String csv_line = Globals.CurrentCompetitionId + ":" + Globals.CurrentMatchNumber + ":" + Globals.CurrentDeviceId;
 
         // If this is NOT a new sequence, we need to write out the previous event id that goes with this one
         if (!in_NewSequence) prev = String.valueOf(seq_number_prev);
 
+        // If the elapsed time is greater than the match length just log the match length
+        double time = Math.round((in_time - Match.startTime) / 10.0) / 100.0;
+        time = Math.min(time, Match.TIMER_AUTO_LENGTH + Match.TIMER_TELEOP_LENGTH);
+        
```

Comment:
why have another variable, when you can just put it all in the "String string_time" line?
like:
String string_time = String.valueof(Math.min(Math.round((in_time - Match.startTime) / 10.0) / 100.0), Match.TIMER_AUTO_LENGTH + Match.TIMER_TELEOP_LENGTH);